lua/entities/ent_jack_claymore/init.lua

```diff
@@ -149,7 +156,7 @@ function ENT:Use( activator )
             local canConstrain = JID.CanConstrain( self, Tr.Entity ) or Tr.Entity:IsWorld()
 
             -- stick into loose mats solidly
-            local isPlantableMat = PlantableMats[Tr.MatType]
+            local isPlantableMat = plantableMats[Tr.MatType]
 
             if isPlantableMat and canConstrain then

```

Comment:
no reason to have this as a variable, you can just directly put this in line 161